src/AddressDriver.sol

```diff
@@ -58,7 +58,7 @@ contract AddressDriver is Managed, ERC2771Context {
     /// @return amt The collected amount
     function collect(IERC20 erc20, address transferTo) public whenNotPaused returns (uint128 amt) {
         amt = dripsHub.collect(_callerUserId(), erc20);
-        erc20.safeTransfer(transferTo, amt);
+        if (amt > 0) dripsHub.withdraw(erc20, transferTo, amt);
```

Comment:
Are we planning to implement a toAddress for the collect? 

It makes writing a correct Driver just harder.
Like if you call `collect` don't forget to call `withdraw` as well afterward otherwise your funds are gone.